### ISSUE TITLE
How to copy from one data type to another data type with same name?

### ISSUE THREAD

**Issue by kumarsiva07** [jinzhu/copier#34]
```
// Model
type User struct {
	JoinedAt   time.Time
	ReleavedAt *time.Time
}

// Resource
type UserResource struct {
	JoinedAt   string
	ReceivedAt string
}
```

Want to copy from User to UserResource.

**Comment by daluu** [comment]
I wonder if that's possible (at the moment), as it would require casting the data types in the copy process.

**Comment by ilius** [comment]
Can you cast `time.Time` to string?
You need to specify the format somehow (unless it's JSON-specific)

**Comment by uded** [comment]
Closing as it is addressed via #115
